_includes/configuration/dockerized-image-proxy.md

```diff
@@ -0,0 +1,92 @@
+### Deploy the TinyMCE image proxy server-side component using Docker (self-hosted enterprise bundle)
+
+The TinyMCE image proxy server-side component can be deployed using Docker or an container orchestration application such as Kubernetes. {{site.companyname}} provides a pre-configured package for creating a Image Proxy Docker image.
+
+> **Warning**: Do not push this docker image to a publicly accessible container registry. Doing so will constitute a breach of the [{{site.companyname}} Self-Hosted Software License Agreement](https://about.tiny.cloud/legal/tiny-self-hosted-software-license-agreement-enterprise/).
+
+#### Procedure
+
+The following procedure covers downloading, configuring, building and testing the {{site.productname}} Image Proxy Docker image.
+
+> **Note**: This procedure assumes that the **{{ site.productname }} Enterprise Bundle** has been extracted as described in [Download and extract the TinyMCE Enterprise Bundle](#downloadandextractthetinymceenterprisebundle).
+
+4. Navigate into the `tinymce-services` folder.
+
+    ```sh
+    cd tinymce-enterprise/tinymce-services/
+    ```
+
+5. Extract the contents of `ephox-hyperlinking-docker-starter-kit.zip`, such as:
+
+    ```sh
+    unzip ephox-hyperlinking-docker-starter-kit.zip -d hyperlinking-service-dockerfile
+    ```
+
+6. Copy `ephox-hyperlinking.war` into the directory containing the extracted files, such as:
+
+    ```sh
+    cp ephox-hyperlinking.war hyperlinking-service-dockerfile/
+    ```
+
+4. Navigate into the `hyperlinking-service-dockerfile` folder.
+
+    ```sh
+    cd hyperlinking-service-dockerfile
+    ```
+5. Set the permissions on the extracted files to executable.
+
+    ```sh
+    chmod +x *.sh
+    ```
+6. _Optional_: Edit the `http` configuration settings in `ephox-image-proxy-docker-env.conf`. The configurable settings are in the `http` section of the file. For example:
+
+    ```
+    http {
+      max-connections = 100
+      max-connections-per-host = 10
+      max-redirects = 10
+      max-retries = 3
+      request-timeout-seconds = 10
+      trust-all-cert = false
+      websphere = {
+        use-ssl-config = true
+      }
+    }
+    ```
+    For information on the `http` configuration setting, see: [Configure server-side components - `http`]({{site.baseurl}}/enterprise/server/configure/#httpoptional).
```

Comment:
You don't mention configuring the image proxy service. I think the parameter is `ephox > image-proxy > size-limit`. 

Speaking of which, the example for image-proxy here is incorrect (https://www.tiny.cloud/docs/enterprise/server/configure/#image-proxyoptional)

Reply:
added the size-limit info